<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TFormDoc</class>
 <widget class="QWidget" name="TFormDoc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>361</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <action name="actOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Pics/images/122.bmp</normaloff>:/Pics/images/122.bmp</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actCut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Pics/images/200.bmp</normaloff>:/Pics/images/200.bmp</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actCopy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Pics/images/202.bmp</normaloff>:/Pics/images/202.bmp</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actPaste">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Pics/images/204.bmp</normaloff>:/Pics/images/204.bmp</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actFont">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Pics/images/506.bmp</normaloff>:/Pics/images/506.bmp</iconset>
   </property>
   <property name="text">
    <string>字体</string>
   </property>
   <property name="toolTip">
    <string>设置字体</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actClose">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Pics/images/132.bmp</normaloff>:/Pics/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="toolTip">
    <string>关闭本窗口</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actUndo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Pics/images/206.bmp</normaloff>:/Pics/images/206.bmp</iconset>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="toolTip">
    <string>撤销编辑操作</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actRedo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Pics/images/208.bmp</normaloff>:/Pics/images/208.bmp</iconset>
   </property>
   <property name="text">
    <string>重做</string>
   </property>
   <property name="toolTip">
    <string>重做编辑操作</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actClose</sender>
   <signal>triggered()</signal>
   <receiver>TFormDoc</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
